zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -1018,9 +1018,40 @@ GROUP BY notes.account_id, notes.txid";
 ///   both outputs received as a consequence of wallet-internal transfers
 ///   and as change.
 ///
-/// The `to_address` column will only contain an address when the recipient is
-/// external. In all other cases, the recipient account id indicates the account
-/// that controls the output.
+/// # Columns
+/// - `txid`: The id of the transaction in which the output was sent or received.
+/// - `output_pool`: The value pool for the transaction; valid values for this are:
+///   - 0: Transparent
+///   - 2: Sapling
+///   - 3: Orchard
+/// - `output_index`: The index of the output within the transaction bundle associated with
+///   the `output_pool` value; that is, within `vout` for transparent, the vector of
+///   Sapling `OutputDescription` values, or the vector of Orchard actions.
+/// - `from_account_uuid`: The UUID of the wallet account that created the output, if the wallet
+///   spent notes in creating the transaction. Note that if multiple accounts in the wallet
+///   contributed funds in creating the associated transaction, redundant rows will exist in the
+///   output of this view, one for each such account.
+/// - `to_account_uuid`: The UUID of the wallet account that received the output, if any; for
+///   outgoing transaction outputs this will be `NULL`.
+/// - `address`: The address to which the output was sent; for received outputs, this is the
+///   address at which the output was received, or `NULL` for wallet-internal outputs.
+/// - `diversifier_index_be`: The big-endian representation of the diversifier index (or, for
+///   transparent addresses, the BIP 44 change-level index of the derivation path) of the receiving
+///   address. This will be `NULL` for outgoing transaction outputs.
+/// - `value`: The value of the output, in zatoshis.
+/// - `is_change`: `0` for outgoing outputs and outputs received at external-facing addresses, `1`
+///   for outputs received at wallet-internal addresses. This represents a best-effort judgement
+///   for whether or not the output should be considered change, and may not be correct for
+///   cross-account internal transactions, shielding transactions, or outputs explicitly sent from
+///   the wallet to itself. The determination of what counts as change is somewhat subjective and
+///   the value of this column should be used with caution.
+/// - `memo`: The binary content of the memo associated with the output, if the output is a
+///   shielded output and the memo was received by the wallet, sent by the wallet or was able to be
+///   decrypted with the wallet's outgoing viewing key.
+///
+/// Wallet-internal outputs will always have the `to_account_uuid` column set to a non-null value.
+/// The `to_address` column will be set to the wallet-internal address in this case, and to the
+/// address to which the output was sent if the recipient is external to the wallet.
```

Comment:
The first half of this sentence is incorrect (for the same reason we needed a `LEFT JOIN addresses`).